c7n/resources/route53.py

```diff
@@ -17,7 +17,9 @@
 
 from c7n.query import QueryResourceManager, ChildResourceManager
 from c7n.manager import resources
-from c7n.utils import chunks, get_retry, generate_arn, local_session
+from c7n.utils import chunks, get_retry, generate_arn, local_session,type_schema
```

Comment:
Small nitpick: inconsistent spacing (missing after last comma).

Reply:
This is fixed.